src/groups/mwc/mwcc/mwcc_orderedhashmap.h

```diff
@@ -100,6 +100,7 @@
 //..
 
 // MWC
+#include <mwcu_safecast.h>
```

Comment:
Might move it to other source since it's not used in the header `mwcc_orderedhashmap.h`